packages/session-replay-browser/src/session-replay.ts

```diff
@@ -334,16 +334,12 @@ export class SessionReplay implements AmplitudeSessionReplay {
       flushMaxRetries: this.config.flushMaxRetries,
       apiKey: this.config.apiKey,
       deviceId: this.getDeviceId(),
-      sampleRate: this.getSampleRate(),
+      sampleRate: this.config.sampleRate,
       serverZone: this.config.serverZone,
-      onComplete: this.cleanUpSessionEventsStore.bind(this),
+      onComplete: this.sessionIDBStore.cleanUpSessionEventsStore.bind(this),
     });
   }
 
-  getSampleRate() {
```

Comment:
I noticed this is an unnecessary wrapper, as we set the sample rate to the DEFAULT_SAMPLE_RATE in the config class constructor: https://github.com/amplitude/Amplitude-TypeScript/blob/v1.x/packages/session-replay-browser/src/config.ts#L41-L41